tests/common/snappi_tests/traffic_generation.py

```diff
@@ -529,8 +530,14 @@ def verify_basic_test_flow(flow_metrics,
         else:
             pytest_assert(tx_frames == rx_frames,
                           "{} should not have any dropped packet".format(metric.name))
-
-            exp_test_flow_rx_pkts = data_flow_config["flow_rate_percent"] / 100.0 * speed_gbps \
+            # Check if flow_rate_percent is a dictionary
+            if isinstance(data_flow_config["flow_rate_percent"], dict):
+                # Sum all the flow rate percentages
+                total_flow_rate_percent = sum(data_flow_config["flow_rate_percent"].values())
```

Comment:
if there are multiple test flow, should we use individual flow rate instead of summary?

Reply:
done